conda_smithy/azure_ci_utils.py

```diff
@@ -0,0 +1,187 @@
+import os
+import typing
+
+from vsts.vss_connection import VssConnection
```

Comment:
Do we need a conda-forge package for this?

Reply:
yes, I've started the process